app/models/address.rb

```diff
@@ -135,8 +135,23 @@ def to_a
   end
 
   def fetch_county_fips_code
-    us_county = BenefitMarkets::Locations::CountyFips.where(state_postal_code: state, county_name: county).first
-    us_county ? us_county.county_fips_code : nil
+    state_abbrev = state.blank? ? '' : state.upcase
+    county_name = county.blank? ? '' : county.titlecase
+    county_fip = ::BenefitMarkets::Locations::CountyFips.where(state_postal_code: state_abbrev, county_name: county_name).first
+
+    if county_fip.present?
+      county_fip.county_fips_code
+    else
+      county_fip_by_zip_state.present? ? county_fip_by_zip_state.county_fips_code : ""
```

Comment:
Is this needed because in some cases we don't store the county name?  In what case would we be looking up the county FIPS without storing the county name?

I'm confused why we need to have a fall back search to ZIP if we can't find by county and state.

Some comments might help clarify this.

Reply:
https://github.com/ideacrew/enroll/pull/1392#discussion_r761512094

Reply:
We always save county name through UI, scenario happens when we import data and data missing county name.